### PR TITLE
Clarified lists and removed unnecessary bold

### PR DESCRIPTION
**Pull request by emilybrown1**
Updating the style of Override-Volumetric-Clouds

### CODE REVIEW

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Remove Texture/Textures.

File: com.unity.render-pipelines.high-definition/Documentation~/Override-Volumetric-Clouds.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ When importing these two map Textures, disable **sRGB**. For best results, do no
...
+| - **Cloud Map Offset**            | The **X** and **Y** UV offset for the cloud map Textures. HDRP uses the **X** and **Y** values to offset the clouds across the sky.<br/>If **Cloud Control** is set to **Advanced**, this affects **Cumulus Map**, **Alto Stratus Map**, **Cumulonimbus Map**, and **Rain Map**.<br/>If **Cloud Control** is set to **Custom**, this affects the Texture assigned to the **Cloud Map** property.<br/><br/>This property only appears if you set **Cloud Control** to **Advanced** or **Custom**, or if you set it to **Simple** and then set **Cloud Preset** to **Custom**. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Override-Volumetric-Clouds.md:89]: Make the start of this sentence the same as the other change you made: 

"The **X** and **Y** UV offset for one or more cloud map Textures."

Consistency is key! :D

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Phrasing!

**Review by Vic-Cooper — APPROVED** [review]
Looks good! :)

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**